src/components/Widgets/TeamNews/TeamNewsWidget.tsx

```diff
@@ -0,0 +1,134 @@
+import { Box } from '@mui/material';
+import { GridColDef } from '@mui/x-data-grid-pro';
+import {
+  EngagementFilters,
+  ProjectStatusLabels,
+  ProjectStatusList,
+} from '~/api/schema.graphql';
+import { EngagementListDocument } from '~/scenes/Projects/List/EngagementList.graphql';
+import { EngagementDataGridRowFragment } from '../../EngagementDataGrid';
+import { enumColumn, textColumn, useDataGridSource } from '../../Grid';
+import { RoleIcon } from '../../RoleIcon';
+import { Link } from '../../Routing';
+import { TableWidget } from '../TableWidget';
+import { WidgetConfigProps } from '../widgetConfig';
+
+export const TeamNewsColumns: Array<GridColDef<EngagementDataGridRowFragment>> =
+  [
+    {
+      headerName: 'Project',
+      field: 'project.name',
+      ...textColumn(),
+      width: 200,
+      flex: 1,
+      valueGetter: (_, row) => row.project.name.value,
+      renderCell: ({ value, row }) => (
+        <Link to={`/projects/${row.project.id}`}>{value}</Link>
+      ),
+      hideable: false,
+    },
+    {
+      headerName: 'Language / Intern',
+      field: 'nameProjectLast',
+      ...textColumn(),
+      width: 200,
+      valueGetter: (_, row) => {
+        return row.__typename === 'LanguageEngagement'
+          ? row.language.value?.name.value
+          : row.__typename === 'InternshipEngagement'
+          ? row.intern.value?.fullName
+          : null;
+      },
+      renderCell: ({ value, row }) => {
+        return row.__typename === 'LanguageEngagement' ? (
+          <Link to={`/languages/${row.language.value!.id}`}>{value}</Link>
+        ) : row.__typename === 'InternshipEngagement' ? (
+          <Link to={`/users/${row.intern.value!.id}`}>{value}</Link>
+        ) : null;
+      },
+      hideable: false,
+      serverFilter: ({ value }) =>
+        ({ engagedName: value } satisfies EngagementFilters),
+    },
+    {
+      headerName: 'Progress',
+      type: 'string',
+      field: 'project.type',
+    },
+    {
+      field: 'project.status',
+      ...enumColumn(ProjectStatusList, ProjectStatusLabels, {
+        orderByIndex: true,
+      }),
+      valueGetter: (_, row) => row.project.status,
+      headerName: 'Status',
+      width: 160,
+    },
+    {
+      headerName: 'Team News',
+      type: 'string',
+      field: 'project.primaryLocation',
+      valueGetter: (_, row) => row.project.primaryLocation.value?.name.value,
+      renderCell: () => (
+        <Box
+          sx={{
+            height: '100%',
```

Comment:
Maybe? 
```css
height : 1
```
i believe this is does the same as `height : '100%'`.  If I am wrong then ignore.